modules/virt-generalizing-linux-vm-image.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_custom/virt-creating-vms-uploading-images.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-generalizing-linux-vm-image_{context}"]
+= Generalizing a VM image
+
+You can generalize a {op-system-base-full} image to remove all system-specific configuration data before you use the image to create a golden image, a preconfigured snapshot of a virtual machine (VM). You can use a golden image to deploy new VMs.
+
+Generalizing a {op-system-base} VM is done using the `virtctl`, `guestfs`, and `virt-sysprep` tools.
+
+.Prerequisites
+
+* An {op-system-base} virtual machine.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. If the {op-system-base} virtual machine is running, stop it.
++
+[source,terminal]
+----
+$ virtctl stop <rhel-vm-name>
+----
+
+. Optional: Clone the virtual machine to avoid losing the data from your original VM. You can then generalize the cloned VM.
+
+. Retrieve the `dataVolume` that stores the root filesystem for the VM by running the following command:
++
+[source,terminal]
+----
+$ oc get vm <my-vm-name> -o jsonpath="{.spec.template.spec.volumes}{'\n'}"
+
+[{"dataVolume":{"name":"my-vm-volume"},"name":"rootdisk"},{"cloudInitNoCloud":{...}]
+----
+
+. Retrieve the Persistent Volume Claim (PVC) that matches the listed `dataVolume` by running the followimg command:
++
+[source,terminal]
+----
+$ oc get pvc
+NAME            STATUS   VOLUME  CAPACITY   ACCESS MODES  STORAGECLASS     AGE
+my-vm-volume Bound  …
+----
+
+[NOTE]
+====
+If your cluster configuration does not enable you to clone a VM, and you want to avoid losing the data from your original VM, you can clone the VM PVC to a data volume instead. You can then use the cloned PVC to create a golden image.
+
+If you are creating a golden image by cloning a PVC, continue with the steps below, using the cloned PVC.
```

Comment:
```suggestion
If you are creating a golden image by cloning a PVC, continue with the next steps.
```
- I would avoid using "below". Consider using the "next steps" or "following steps".
- "using a cloned PVC" is redundant. "...creating a golden image **by cloning a PVC**" already gives context.

Reply:
I don't feel that "using the cloned PVC" is necessarily redundant, since the user could mistake needing to use the original PVC in the next steps. I'd prefer to keep that bit for clarity as well. I did change "below".